lib/WP_Auth0_LoginManager.php

```diff
@@ -162,6 +149,12 @@ public function init_auth0() {
       return;
     }
 
+    // Check for valid state UUID
+    $state_decoded = json_decode( base64_decode( $this->query_vars( 'state' ) ), TRUE );
```

Comment:
what if `state` is missing from `query_vars`. How will the code behave?

Reply:
Then `empty( $state_decoded[ 'nonce' ] )` will be true and the login process will fail

Reply:
So you are saying this calls `json_decode( base64_decode( $this->query_vars( 'state' ) ), TRUE );` don't fail when the `query_vars.state` is missing/null/undefined, right?

Reply:
Exactly. `$this->query_vars( 'state' )` returns `null` if not found, `base64_decode()` returns `false` in that case, and `json_decode()` returns `false`. PHP's `empty()` returns `true` even if that key does not exist (no errors).

Reply:
Perfect, thanks.